src/main/java/io/datanerds/avropatch/value/DefaultSchema.java

```diff
@@ -81,4 +81,11 @@
                     + "        \"logicalType\": \"uuid\"\n"
                     + "    }]\n"
                     + "}";
+    String TIMESTAMP =
```

Comment:
As Frank elaborated earlier, not as easy since it needs to be a constant. When you are able to access java's file api it will already be to late. The only real workaround is having a file and a generator which pre compilation generates a java file containing the schema.